libdevcore/Log.h

```diff
@@ -167,17 +156,17 @@ class LogOutputStreamBase
 	void append(u160 const& _t) { m_sstr << EthNavy << _t << EthReset; }
 	void append(double _t) { m_sstr << EthBlue << _t << EthReset; }
 	template <unsigned N> void append(FixedHash<N> const& _t) { m_sstr << EthTeal "#" << _t.abridged() << EthReset; }
-	void append(h160 const& _t) { m_sstr << EthRed "@" << _t.abridged() << EthReset; }
-	void append(h256 const& _t) { m_sstr << EthCyan "#" << _t.abridged() << EthReset; }
-	void append(h512 const& _t) { m_sstr << EthTeal "##" << _t.abridged() << EthReset; }
+	void append(h160 const& _t) { m_sstr << EthRed "@" << _t.h160::abridged() << EthReset; }
```

Comment:
That was a partial attempt to resolve templates manually, but I don't do this in this PR.